EIPS/eip-sc.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: <to be assigned>
+title: Interface for Security Contact of Smart Contract
+description: An interface for security notice using asymmetric encryption
+author: Zainan Zhou (@xinbenlv)
+discussions-to: https://ethereum-magicians.org/t/erc-interface-for-security-contract/10303
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-09
+requires: 165
+---
+
+An interface for smart contract developer to specify to communicate to them in secured way.
+A typical use case is for security researcher to report to developer the
+security vulnerability.
+
+## Abstract
+An interface for security notice using asymmetric encryption. The interface exposes a asymmetric encryption key
+and a destination of delivery.
```

Comment:
```suggestion
## Abstract

An interface for smart contracts to specify how to communicate with their developers using asymmetric encryption. For example, a security researcher may wish to report vulnerabilities over a secured channel. The interface exposes an asymmetric encryption key and a destination for delivery.
```

Reply:
Thank you, update according to your revision.